src/Core/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -73,7 +73,7 @@ public class MsSqlQueryExecutor : QueryExecutor<SqlConnection>
                   httpContextAccessor)
         {
             RuntimeConfig runtimeConfig = runtimeConfigProvider.GetConfig();
-            IEnumerable<KeyValuePair<string, DataSource>> mssqldbs = runtimeConfig.GetDataSourceNamesToDataSourcesIterator().Where(x => x.Value.DatabaseType == DatabaseType.MSSQL);
+            IEnumerable<KeyValuePair<string, DataSource>> mssqldbs = runtimeConfig.GetDataSourceNamesToDataSourcesIterator().Where(x => x.Value.DatabaseType == DatabaseType.MSSQL || x.Value.DatabaseType == DatabaseType.DWSQL);
```

Comment:
as per my understanding session context is supported https://learn.microsoft.com/en-us/sql/t-sql/functions/session-context-transact-sql?view=azure-sqldw-latest. I did my integrated test with a dw using an identity token so that is supported for sure.